fiddler/tests/conftest.py

```diff
@@ -1,15 +1,31 @@
+import os
+
 import pytest
 
+FIDDLER_API_KEY = os.environ.get('FIDDLER_API_KEY')
+
 
 @pytest.fixture(scope='session')
-def dd_environment():
-    yield
+def dd_environment(instance, e2e_instance):
+    if FIDDLER_API_KEY:
+        yield e2e_instance
+    else:
+        yield instance
 
 
-@pytest.fixture
+@pytest.fixture(scope='session')
 def instance():
     return {
         "url": "https://demo.fiddler.ai",
-        "fiddler_api_key": "K4ph7ORDcIO2xVIEA6KxL1o1zHjZockgurhCOZOUSVs",
+        "fiddler_api_key": "apikey",
+        "organization": "demo",
+    }
```

Comment:
This instance will run in the CI with a fake API key

Reply:
Looks good.